examples/diffantom/sticks2dwis

```diff
@@ -160,6 +163,8 @@ for i in range(len(vfs_im)):
 
 print("\tReading acquisition sequence...")
 bvals = np.loadtxt(args.bvals)
+if (np.max(bvals) < 1000) or (np.max(bvals) > 10000) :
+    warnings.warn("The maximum b-value is not in the range 1000 to 10000 [s\ mm\ :superscript:`-2`].")
```

Comment:
Yes, just to be clear in the message you can recall that the expected unit is [s/mm^2].